cmd/telem_view/telem_view.go

```diff
@@ -110,7 +135,22 @@ func main() {
 				data := p.Data()
 
 				offset := 0
+
+				// prep slices to collect all updates for this packet
+				countMeas := len(pkt.Measurements)
+				valStrs := make([]string, countMeas)
+				hexStrs := make([]string, countMeas)
+				binStrs := make([]string, countMeas)
+
+				// capture flags locally tso we avoid closure/capture races
+				hexLocal := hexOn
+				binLocal := binOn
+
 				for i, name := range pkt.Measurements {
+					valStrs[i] = padValue("–")
+					hexStrs[i] = ""
+					binStrs[i] = ""
```

Comment:
I believe this is unnecessary because the slice is already zero-initialized. Not sure if go is smart enough to optimize this.